src/Computer.elm

```diff
@@ -0,0 +1,17 @@
+module Computer exposing (..)
+
+import Board exposing (Board, Mark(..), toArray)
+import Array exposing (..)
+
+chooseMove : Board -> Int
+chooseMove board =
+  let
+    indexedBoard = Array.indexedMap (,) (toArray board)
+    availableMoves = Array.filter (\elem -> snd elem == Nothing) indexedBoard
+  in
+    case (get 0 availableMoves) of
+     Just (a, b) -> a
+     Nothing -> -1
```

Comment:
This immediately makes me consider returning a `Maybe` type from this function. Did you consider that? Pros cons of `Int` vs `Maybe Int` here?